### PR TITLE
fix table line length, remove #noqa, use grid

### PR DESCRIPTION
**Pull request by mikofski**
- [ ] Closes #xxxx
 - [ ] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [ ] Tests added
 - [ ] Updates entries to [`docs/sphinx/source/api.rst`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/api.rst) for API changes.
 - [ ] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/master/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [ ] Pull request is nearly complete and ready for detailed review.
 - [ ] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by mikofski** [comment]
I found a few typos in the original table:
mintues -> minutes
Relatitudeive -> Relative
phenonema -> phenomena

I also checked the html source code from the rendered documents are exactly the same, minus the typo changes.

**Comment by wholmgren** [comment]
Thanks @mikofski. I have a mild preference for this format. What do others think? @cwhanse?

Did you do this manually or do you have a slick way of doing it?

**Comment by mikofski** [comment]
**TL;DR**: I did it manually

After a little research I found some interesting options ...
<details>

1. you can import a csv data or file directly into an rst file and docutils will generate a table for you:

    * or csv data:
        ```rst
        .. csv-table:: Frozen Delights!
           :header: "Treat", "Quantity", "Description"
           :widths: 15, 10, 30

           "Albatross", 2.99, "On a stick!"
           "Crunchy Frog", 1.49, "If we took the bones out, it wouldn't be
           crunchy, now would it?"
           "Gannet Ripple", 1.99, "On a stick!"
        ```
    * with csv file (_warning_ not recommended by docutils due to vulnerability?):
        ```rst
        .. csv-table:: Table Title
           :file: CSV file path and name
           :widths: 30, 70
           :header-rows: 1
        ```
    * see [docutils](https://docutils.sourceforge.io/docs/ref/rst/directives.html#id4), [stackoverflow](https://stackoverflow.com/a/46260528/1020470), and [this rando website](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#csv-files)

2. the docutils list table directives:

    * example:
        ```rst
        .. list-table:: Frozen Delights!
           :widths: 15 10 30
           :header-rows: 1

           * - Treat
             - Quantity
             - Description
           * - Albatross
             - 2.99
             - On a stick!
           * - Crunchy Frog
             - 1.49
             - If we took the bones out, it wouldn't be
               crunchy, now would it?
           * - Gannet Ripple
             - 1.99
             - On a stick!
        ```
    * [docutils list table](https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table)
    * [rando website again](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#list-table-directive)

2. nifty sublime text table package (unsupported since 2015):

    * [package control table editor](https://packagecontrol.io/packages/Table%20Editor)
    * [github repo](https://github.com/vkocubinsky/SublimeTableEditor)

3. slick riv vim plugin (current as of 2018):

    * [github repo](https://github.com/gu-fan/riv.vim) and [instantRst addon](https://github.com/gu-fan/InstantRst)
</details>

**Comment by cwhanse** [comment]
I don't have a preference for the old (long-line) or new format, as far as reading the information in the source code.

**Comment by wholmgren** [comment]
Ok, let's go for it. Thanks @mikofski!
